pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -128,20 +137,33 @@
 @Api(tags = Constants.SEGMENT_TAG)
 @Path("/")
 public class PinotSegmentRestletResource {
-  private static Logger LOGGER = LoggerFactory.getLogger(PinotSegmentRestletResource.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(PinotSegmentRestletResource.class);
+
+  @Inject
+  ControllerConf _controllerConf;
 
   @Inject
   PinotHelixResourceManager _pinotHelixResourceManager;
 
+  @Inject
+  Executor _executor;
+
+  @Inject
+  HttpConnectionManager _connectionManager;
+
+  @Inject
+  ControllerMetrics _controllerMetrics;
+
+
   @GET
   @Produces(MediaType.APPLICATION_JSON)
   @Path("/segments/{tableName}")
   @ApiOperation(value = "List all segments", notes = "List all segments")
   public List<Map<TableType, List<String>>> getSegments(
-      @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
-      @ApiParam(value = "OFFLINE|REALTIME") @QueryParam("type") String tableTypeStr) {
+          @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
```

Comment:
there seems to be some unnecessary formatting/whitespaces on all methods of this file. Could you revert those? you should be able to auto-format using the IDE
same comment for TablesResourceTest file

Reply:
Some setting on my IDE got changed for the Pinot checkstyle. Have updated it again. Should be fixed in my next commit.